src/SDKs/Peering/Management.Peering.Tests/Peering.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Peering.Tests</PackageId>
+    <AssemblyName>Peering.Tests</AssemblyName>
+    <Version>0.8.0-preview</Version>
```

Comment:
Test csproj must be versioned to `1.0.0` and never modified

Reply:
done